docs/technical-documentation.md

```diff
@@ -7,50 +7,50 @@ category: Technical documentation
 
 ### Network store server
 
-Almost all of the feature developed in PowSyBl relies on IIDM API for accessing network data, the idea of the service is to re-implement the IIDM API to be better integrated in a micro-services typical architecture. Instead of an in-memory implementation like in PowSyBl core repository, this implementation is backed by a Cassandra database. Network model is stored in the database in a detailed way (one table per equipment), so that we can query only the amount of needed data and ensure a good performance for a simple switch position change (few data), a single line diagram rendering (substation wide data) or a loadflow run (network wide data). Network data should only be accessed from this REST Web service using the provided Java client (the one which implement IIDM API) and never directly from the REST interface.
+Almost all the features developed in PowSyBl rely on the IIDM API for accessing network data. The idea of this service is to re-implement the IIDM API to be better integrated in a typical microservice architecture. Instead of an in-memory implementation like in the PowSyBl core repository, this implementation is backed by a Cassandra database. The network model is stored in the database in a structured way (one table per equipment type and with indexes), so that we can query only the needed data and ensure good performance for common operations. Examples are a simple switch position change (only a few values), a single line diagram rendering (substation wide data) or a loadflow run (network wide data). The REST interface exposed by this service is very low level for performance reasons. In particular, the write operations do not prevent inconsistent modifications. Therefore, network data should usually be accessed from this service using the provided Java client (the one which implements the IIDM API) and not directly from the REST interface because the java client is safe and prevents inconsistent modifications.
 
 ### Geographical data server
 
-Substations and lines path positions can be stored and queried using this geographical data server. A Cassandra database is used to store the data. Data can be indexed by countries and nominal voltages. This Web service is typically used to display a network on a map.
+Substations and line paths positions can be stored and queried using this geographical data server. A Cassandra database is used to store the data. Data can be indexed by countries and nominal voltages. This service is typically used to display a network on a map. This service abstracts away the different sources of geographical data.
 
 ### Network conversion server
 
-This Web service is responsible for the connection between a case server and a network store server. It allows converting a case whatever the format it is stored to an IIDM network in the network store server.
+This service can convert a network stored in a case server to an IIDM network stored in the network store server.
```

Comment:
Maybe you should repeat again here that the case server stored the network in its original format ?

Reply:
I think that this fact documents the behavior of the case server and therefore should not be present in the Network conversion server section